### PR TITLE
Added SDFtoGCSOperator

### PR DESCRIPTION
**Pull request by michalslowikowski00**
Added handling SDF entities (creating operation task and waiting for operation task) and saving in Google Cloud Storage.

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/providers/google/marketing_platform/example_dags/example_display_video.py [hunk 1]
```diff
@@ -19,22 +19,34 @@
...
 OBJECT_NAME = os.environ.get("GMP_OBJECT_NAME", "files/report.csv")
+PATH_TO_UPLOAD_FILE = os.environ.get(
+    "GCP_GCS_PATH_TO_UPLOAD_FILE", "test-gcs-example.txt"
+)
+PATH_TO_SAVED_FILE = os.environ.get(
+    "GCP_GCS_PATH_TO_SAVED_FILE", "test-gcs-example-download.txt"
+)
+BUCKET_FILE_LOCATION = PATH_TO_UPLOAD_FILE.rpartition("/")[-1]
+SDF_VERSION = os.environ.get("GMP_SDF_VERSION", "SDF_VERSION_5_1")
+BQ_DATASET = "airflow_test"
```
Comment (mik-laj) [inline airflow/providers/google/marketing_platform/example_dags/example_display_video.py]: Can you parameterize it via env var?

File: airflow/providers/google/marketing_platform/hooks/display_video.py [hunk 2]
```diff
@@ -170,3 +184,51 @@ def download_line_items(self, request_body: Dict[str, Any]) -> List[Any]:
             .execute(num_retries=self.num_retries)
         )
         return response["lineItems"]
+
+    def create_sdf_download_operation(self, body_request: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates an SDF Download Task and Returns an Operation.
+        :param body_request: Body request.
```
Comment (mik-laj) [inline airflow/providers/google/marketing_platform/hooks/display_video.py:192]:
```suggestion

        :param body_request: Body request.
```

Comment (mik-laj) [reply airflow/providers/google/marketing_platform/hooks/display_video.py:192]: The documentation will not be built without it.

File: tests/providers/google/marketing_platform/hooks/test_display_video.py [hunk 3]
```diff
@@ -239,3 +257,163 @@ def test_upload_line_items_should_return_equal_values(self, get_conn_mock):
...
+    )
+    def test_download_media_called_once_with_params(self, get_conn_to_display_video):
+        resource_name = "resource_name"
+
+        self.hook.download_media(resource_name=resource_name)
+        get_conn_to_display_video.return_value.\
+            media.return_value.\
+            download_media.assert_called_once_with(resource_name=resource_name)
+
+    # @mock.patch(
```
Comment (mik-laj) [inline tests/providers/google/marketing_platform/hooks/test_display_video.py]: ?

File: airflow/providers/google/marketing_platform/hooks/display_video.py (hunk 2 above)
Comment (michalslowikowski00) [reply airflow/providers/google/marketing_platform/hooks/display_video.py:192]: Hm... Ok, thanks. 
I did not have issues when I built documentation locally. I will fix it.

File: tests/providers/google/marketing_platform/hooks/test_display_video.py (hunk 3 above)
Comment (michalslowikowski00) [reply tests/providers/google/marketing_platform/hooks/test_display_video.py]: Done.

File: airflow/providers/google/marketing_platform/example_dags/example_display_video.py (hunk 1 above)
Comment (michalslowikowski00) [reply airflow/providers/google/marketing_platform/example_dags/example_display_video.py]: Ok.

**Review by turbaszek — APPROVED** [review]

**Pull request merged**